src/apis/getItem.ts

```diff
@@ -0,0 +1,10 @@
+import { API_KEY } from '@/constants/constantValues';
+
+export const getItem = (contentType: string, id: string) => {
```

Comment:
Missing TS return type.

Reply:
Can `contentType` be more specific? Not just string?

Reply:
`contentType` can be more specific to restrict the input to only specific values. I will implement this as well as missing TS return type.